index.html

```
@@ -127,77 +127,142 @@ <h2>Dictionary {{RTCRtpEncodingParameters}} Members</h2>
-          negotiated by Offer/Answer, resulting in an error if the requested
-          {{RTCRtpEncodingParameters/scalabilityMode}} value is outside this envelope.
+          can be used to remove the H.264/AVC codec from the Offer, improving the
+          chances that a codec supporting temporal scalability is negotiated.
         </p>
         <p>
           When {{RTCRtpTransceiverInit/sendEncodings}} is used to
           request the sending of multiple simulcast streams using
-          {{RTCPeerConnection/addTransceiver()}}, it is not possible to
-          configure the sending of "S" scalability modes. The browser
-          may only be configured to send simulcast encodings with
-          multiple SSRCs and RIDs, or alternatively, to send all
-          simulcast encodings on a single RTP stream. The
-          <a>user agent</a>  MUST return {{OperationError}} in
-          {{RTCRtpSender/setParameters()}} or
-          {{RTCPeerConnection/addTransceiver()}} in response to
-          an attempt to simultaneously utilize both simulcast transport
-          techniques.
+          {{RTCPeerConnection/addTransceiver()}}, an "S mode" cannot
+          be requested. The browser may only be configured to send
+          simulcast encodings with multiple SSRCs and RIDs, or
+          alternatively, to send all simulcast encodings on a single
+          RTP stream. Simultaneously using both simulcast transport
+          techniques is not permitted. 
         </p>
+      </section>
+      <section id="setparameters">
+       <h3>{{RTCRtpSender/setParameters()}}</h3>
+         <p>
+           [[!WEBRTC]] Section 5.2 describes validation of <var>parameters</var>
+           within {{RTCRtpSender/setParameters()}}. Add the following to the 
+           conditions (step 4 within step 6) for an {{InvalidModificationError}}:
+           <ol>
+             <li>
+               <var>encodings</var> contains an encoding whose
```

Throwing and invalid error may be misleading for a developer, which may expect that if the method doesn't throw, then the configuration is expected to be applied. 

I think it is better to throw an error only when the `scalabilityMode` value is not a listed in https://www.w3.org/TR/webrtc-svc/#scalabilitymodes*

---

What if a `scalabilityMode` is defined but not supported for any of the implemented codecs?   Is it better to throw an error immediately, or to act the same as if the `scalabilityMode` was valid for some codec, but not the chosen one?

---

but if we don't throw the dev may think that the configuration is correctly applied and be surprised because it is later  ot in use.

I think it is better to have a consistent API and do not provide any warranty about if the mode is going to be supported at the end or not.

---

I don't think filtering against the spec is good, because modes may be added that haven't made it to the spec yet, and platforms may not have been updated to the latest spec version. I'd suggest throwing on any mode that the platform does not understand, and leave it at that.

---

throwing when the scalability mode is not understood works for me too